README.md

```diff
@@ -1,91 +1,37 @@
-# Starter Kit
+## Tender Management System
+
+### Prerequisites
+  Ensure the following are installed on your system before proceeding:
+
+  - Docker Engine
+  - Docker Compose
+  - Node.js and npm (Node Package Manager)
+
+
+### ⏬ Installation and setup
+
+  To run this website locally, follow the following instructions:
+
+1. Clone the repo
+   ```sh
+   git clone https://github.com/LauraSantiag0/Love-Me-Tender
+   cd Love-Me-Tender
+   ```
+2. Run this command
```

Comment:
What does this command do?

Reply:
Just to make sure you are in right directory.

Reply:
I think this commands starts the database, doesn't it? If that's the case, maybe this would be better:

```
2. Run this command to start the database
```